pkg/apiserver/appcat/appcat.go

```diff
@@ -0,0 +1,38 @@
+package appcat
+
```

Comment:
General nitpick: I prefer not to unnecessarily put all go code under `pkg`. But it's also fine if we keep that

Reply:
I know that generally the code should go in `pkg` but it's fine I guess to remove it.

Reply:
Well AFAIK people started putting code under `pkg` and the main under `cmd` because they saw kubernetes doing it. But IMO there is no good reason to do that in general if you don't have a very large project with a lot of non Go files.

But again this very nipicky so :shrug: